assets/src/app/videos/videos.component.ts

```diff
@@ -138,10 +143,22 @@ export class VideosComponent {
   }
 
   start_process(video): void {
-    this.videoService.ingest(video.legacy_id)
-    .subscribe(response => {
-      // console.log(response);
+    let dialogRef = this.dialog.open(WorkflowDialogComponent, {
+      data: {}
     });
+
+    dialogRef.afterClosed().subscribe(steps => {
```

Comment:
⚠️  `afterClosed()` seems to be called each time time the dialog is closed (even when the user cancels or click outside the dialog), so a POST is sent to create a workflow each time!